docs/monthly-meeting/2022-07.md

```diff
@@ -0,0 +1,149 @@
+# Documentation Community Team Meeting (July 2022)
+
+- **Date:** 2022-07-11
+- **Time:** [21:00 UTC]<!--(https://arewemeetingyet.com/UTC/2022-07-??/21:00/Docs%20Meeting)-->
```

Comment:
```suggestion
- **Time:** [19:00 UTC]<!--(https://arewemeetingyet.com/UTC/2022-07-??/21:00/Docs%20Meeting)-->
```

Am I remember wrong, or wasn't it at 19:00 UTC, instead of our normal 20:00 UTC to account for DST/summer time in most time zones?

Reply:
It was at 21:00 UTC, midnight my time... 19:00 UTC would be better for me :)

Reply:
Ah, right, I miscalculated again...my fault. I'm fine with moving it earlier, but we would want to do another WhenIsGood/Doodle/etc. poll again to capture the best time for everybody.